geonode/proxy/utils.py

```diff
@@ -45,6 +50,10 @@ def unregister_host(self, host):
         self.proxy_allowed_hosts.remove(host)
 
     def get_proxy_allowed_hosts(self):
+        if self._last_registry_load is None or (now() - self._last_registry_load).days >= getattr(
```

Comment:
@mattiagiupponi instead of doing a `getattr` to read the `PROXY_RELOAD_REGISTRY_THRESHOLD_DAYS` on each request it can be read once when the registry I instantiated.
Every small optimization helps with this proxy....